Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -323,6 +323,8 @@ void AVIDump::CloseVideoFile()
   av_frame_free(&s_scaled_frame);
 
   avcodec_free_context(&s_codec_context);
+
+  avio_closep(&s_format_context->pb);
```

Comment:
Hmm, I'm not sure I follow.  Why would you want this when the pointer was allocated?  The point I was trying to make was we probably want to make sure that the allocation actually happened.